docs/source/contributing/reference/manim_directive.rst

```diff
@@ -1,4 +1,4 @@
-﻿manim\_directive
+manim\_directive
```

Comment:
unrelated change?

Reply:
I believe that was autogenerated for some odd reason, when opening rst files.